Mathlib/RingTheory/HahnSeries/Multiplication.lean

```diff
@@ -186,8 +186,9 @@ instance instSMul : SMul (HahnSeries Γ R) (HahnModule Γ' R V) where
             { a : Γ' | (VAddAntidiagonal x.isPWO_support
               ((of R).symm y).isPWO_support a).Nonempty } := by
           intro a ha
+          simp only [Set.mem_setOf_eq]
```

Comment:
I'm not sure if this change is an improvement. It's nice to know that it works, though... does downgrading contrapose! help?